sdk/include/reactos/wine/hid.h

```diff
@@ -0,0 +1,230 @@
+/*
```

Comment:
I wonder, if this file is used / going to be used only within dinput (so far it would appear so, but I don't know whether other Wine-imported modules will use it later), whether it can be placed in dll/directx/wine/dinput.